src/etc/inc/filter.lib.inc

```diff
@@ -302,7 +302,7 @@ function filter_core_rules_system($fw, $defaults)
     // block bogons and private nets
     $bogontmpl = ['type' => 'block', 'log' => !isset($config['syslog']['nologbogons']), 'disablereplyto' => 1];
     $privtmpl = ['type' => 'block', 'log' => !isset($config['syslog']['nologprivatenets']),
-      'from' => '10.0.0.0/8,127.0.0.0/8,100.64.0.0/10,172.16.0.0/12,192.168.0.0/16',
+      'from' => '10.0.0.0/8,127.0.0.0/8,100.64.0.0/10,172.16.0.0/12,169.254.0.0/16,192.168.0.0/16',
```

Comment:
This IPv4 addition isn’t overly popular. I thought about making this list configurable via GUI, but disabling it is always an option. To some degree this is private but it’s also a bit more public than it isn’t and easily configurable.

Reply:
Sorry I didn't pinpoint the subnet when I wrote this. I'm ok with a separate PR. I was reminded of earlier requests to include CG-NAT subnet here.

Reply:
Your comment was about 100.64.0.0/10? Block it in bogons instead of in private networks? Or not block it at all? Either way, I agree this should be a separate PR.

You're okay with blocking 169.254.0.0/16 in private networks and removing it from the bogons here in this PR?